lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -57,22 +58,18 @@ def self.adapter_port
           @adapter_port ||= Datadog::Contrib::Rails::Utils.adapter_port
         end
 
-        def self.tracer
```

Comment:
So normally as this is quite internal, nobody was using this. I let you decide wether it's worth mentioning in the release notes, but I'd say no.